reference_designs/ipu-xrt/matrix_multiplication_array/test.cpp

```diff
@@ -229,17 +228,22 @@ int main(int argc, const char *argv[]) {
   if (VERIFY) {
     std::vector<C_DATATYPE> output_ref0;
     for (uint32_t i = 0; i < C_VOLUME; i++)
-      output_ref0.push_back(0);
-    matmul(AVec, BVec, output_ref0);
+      output_ref0.push_back(K);
```

Comment:
Are we initializing with K?

Reply:
Instead of running the GEMM on the CPU, comparing the output to a single value